ci-operator/step-registry/dpf/hypervisor/deploy-cluster/dpf-hypervisor-deploy-cluster-commands.sh

```
@@ -102,13 +107,35 @@ REMOTE_WORK_DIR="${REMOTE_MAIN_WORK_DIR}/openshift-dpf-${datetime_string}"
+if ssh ${SSH_OPTS} root@${REMOTE_HOST} "ls -ltr; \
+   env; \
+   cd ${REMOTE_WORK_DIR}/openshift-dpf; \
+   git status"; then
   echo "Git repository verified successfully"
 else
   echo "ERROR: Failed to verify git repository at ${REMOTE_WORK_DIR}/openshift-dpf"
   exit 1
 fi
 
+echo "Verify last-openshift-dpf-dir.sh file exists..."
+if ! ssh ${SSH_OPTS} root@${REMOTE_HOST} "test -f ${REMOTE_LAST_OPENSHIFT_DPF_DIR_LOCATION}"; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)